client/js/table-view.js

```diff
@@ -40,7 +40,9 @@ class TableView {
 	}
 
 	renderTableHeader() {
```

Comment:
Set your text editor to use 2-space tabs instead of 4-space tabs; ask me if you have any trouble. This is key for readability, especially when you checked your code into a central repo that other devs will work off of.